license-maven-plugin/src/main/java/com/mycila/maven/plugin/license/document/DocumentType.java

```diff
@@ -28,87 +28,243 @@
+
+  /** The coldfusion component. */
   COLDFUSION_COMPONENT("cfc", HeaderType.DYNASCRIPT3_STYLE),
+
+  /** The coldfusion ml. */
   COLDFUSION_ML("cfm", HeaderType.DYNASCRIPT3_STYLE),
+
+  /** The cpp. */
   CPP("cpp", HeaderType.JAVADOC_STYLE),
+
+  /** The csharp. */
   CSHARP("cs", HeaderType.JAVADOC_STYLE),
+
+  /** The css. */
   CSS("css", HeaderType.JAVADOC_STYLE),
+
+  /** The delphi. */
   DELPHI("pas", HeaderType.BRACESSTAR_STYLE),
+
+  /** The dockerfile. */
   DOCKERFILE("Dockerfile", HeaderType.SCRIPT_STYLE),
+
+  /** The doxia apt. */
   DOXIA_APT("apt", HeaderType.DOUBLETILDE_STYLE),
+
+  /** The doxia faq. */
   DOXIA_FAQ("fml", HeaderType.XML_STYLE),
+
+  /** The dtd. */
   DTD("dtd", HeaderType.XML_STYLE),
+
+  /** The editorconfig. */
   EDITORCONFIG(".editorconfig", HeaderType.SCRIPT_STYLE),
+
+  /** The eiffel. */
   EIFFEL("e", HeaderType.DOUBLEDASHES_STYLE),
+
+  /** The erlang. */
   ERLANG("erl", HeaderType.PERCENT3_STYLE),
+
+  /** The erlang header. */
   ERLANG_HEADER("hrl", HeaderType.PERCENT3_STYLE),
+
+  /** The fortran. */
   FORTRAN("f", HeaderType.EXCLAMATION_STYLE),
+
+  /** The freemarker. */
   FREEMARKER("ftl", HeaderType.FTL),
+
+  /** The groovy. */
   GROOVY("groovy", HeaderType.SLASHSTAR_STYLE),
+
+  /** The gsp. */
   GSP("GSP", HeaderType.XML_STYLE),
+
+  /** The h. */
   H("h", HeaderType.JAVADOC_STYLE),
+
+  /** The haml. */
   HAML("haml", HeaderType.HAML_STYLE),
+
+  /** The htm. */
   HTM("htm", HeaderType.XML_STYLE),
+
+  /** The html. */
   HTML("html", HeaderType.XML_STYLE),
+
+  /** The java. */
   JAVA("java", HeaderType.SLASHSTAR_STYLE),
+
+  /** The javafx. */
   JAVAFX("fx", HeaderType.SLASHSTAR_STYLE),
+
+  /** The javascript. */
   JAVASCRIPT("js", HeaderType.SLASHSTAR_STYLE),
+
+  /** The jsp. */
   JSP("jsp", HeaderType.DYNASCRIPT_STYLE),
+
+  /** The jspx. */
   JSPX("jspx", HeaderType.XML_STYLE),
+
+  /** The kml. */
   KML("kml", HeaderType.XML_STYLE),
+
+  /** The kotlin. */
   KOTLIN("kt", HeaderType.SLASHSTAR_STYLE),
+
+  /** The lisp. */
   LISP("el", HeaderType.EXCLAMATION3_STYLE),
+
+  /** The lua. */
   LUA("lua", HeaderType.LUA),
+
+  /** The mustache. */
   MUSTACHE("mustache", HeaderType.MUSTACHE_STYLE),
+
+  /** The mvel. */
   MVEL("mv", HeaderType.MVEL_STYLE),
+
+  /** The mxml. */
   MXML("mxml", HeaderType.XML_STYLE),
+
+  /** The perl. */
   PERL("pl", HeaderType.SCRIPT_STYLE),
+
+  /** The perl module. */
   PERL_MODULE("pm", HeaderType.SCRIPT_STYLE),
+
+  /** The php. */
   PHP("php", HeaderType.PHP),
+
+  /** The pom. */
   POM("pom", HeaderType.XML_STYLE),
+
+  /** The properties. */
   PROPERTIES("properties", HeaderType.SCRIPT_STYLE),
+
+  /** The python. */
   PYTHON("py", HeaderType.SCRIPT_STYLE),
+
+  /** The ruby. */
   RUBY("rb", HeaderType.SCRIPT_STYLE),
+
+  /** The scala. */
   SCALA("scala", HeaderType.SLASHSTAR_STYLE),
+
+  /** The scaml. */
   SCAML("scaml", HeaderType.HAML_STYLE),
+
+  /** The scss. */
```

Comment:
please don't add javadoc if you don't document.... This does not bring any information.